packages-next/fields/src/types/relationship/views/cards/index.tsx

```diff
@@ -123,125 +133,150 @@ export function Cards({
+                          {errorMessage}
+                        </FieldContainer>
+                      );
+                    }
+                    itemForField[graphqlField] = fieldGetter.data;
+                  }
+                  return (
+                    <field.views.CardValue
+                      key={fieldPath}
+                      field={field.controller}
+                      item={itemForField}
+                    />
+                  );
+                })}
+              </Stack>
+              <Stack across gap="small" marginTop="xlarge">
+                {field.display.inlineEdit && onChange !== undefined && (
+                  <Button
+                    size="small"
+                    disabled={onChange === undefined}
+                    onClick={() => {
+                      onChange({
+                        ...value,
+                        itemsBeingEdited: new Set([...value.itemsBeingEdited, id]),
+                      });
+                    }}
+                    tone="active"
+                  >
+                    Edit
+                  </Button>
+                )}
+                {field.display.removeMode === 'disconnect' && onChange !== undefined && (
+                  <Tooltip content="This item will not be deleted. It will only be removed from this field.">
+                    {props => (
+                      <Button
+                        size="small"
+                        disabled={onChange === undefined}
+                        onClick={() => {
+                          const currentIds = new Set(value.currentIds);
+                          currentIds.delete(id);
+                          onChange({
+                            ...value,
+                            currentIds,
+                          });
+                        }}
+                        {...props}
+                        tone="negative"
+                      >
+                        Remove
+                      </Button>
+                    )}
+                  </Tooltip>
+                )}
+                {field.display.linkToItem && (
+                  <Button
+                    size="small"
+                    weight="link"
+                    tone="active"
+                    css={{ textDecoration: 'none' }}
+                    as={Link}
+                    href={`/${foreignList.path}/${id}`}
+                  >
+                    View {foreignList.singular} details
+                  </Button>
+                )}
+              </Stack>
+            </CardContainer>
+          );
+        })}
+      </Stack>
       {onChange === undefined ? null : field.display.inlineConnect && showConnectItems ? (
         <CardContainer mode="edit">
-          <Stack gap="small" marginY="medium" across>
+          <Stack
+            gap="small"
+            marginY="medium"
+            across
+            css={{
+              width: '100%',
```

Comment:
Same fix as in the images and file field. 
We don't have a good way to individually alter the flex styles of children since each child is programatically wrapped in an element, so we have to do it here.